src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -307,6 +307,7 @@ internal void SendInvocationRequest(ScriptInvocationContext context)
                     {
                         FunctionId = functionMetadata.FunctionId,
                         InvocationId = context.ExecutionContext.InvocationId.ToString(),
+                        CorrelationId = Activity.Current.Id,
```

Comment:
Add unit test in languaworkerchannel tests that verifies CorrelationId is being set